.release-please-manifest.json

```diff
@@ -8,4 +8,5 @@
   "providers/unleash": "0.0.4-alpha",
   "providers/flipt": "0.0.2",
   "providers/configcat": "0.0.3"
```

Comment:
add comma at the end

Reply:
thanks & a good one :) 

updated and should be good to go